src/App.tsx

```diff
@@ -1,10 +1,65 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+
+import { Todo } from './types/Todo';
+
+const USER_ID = 10613;
+
+enum FilterStatus {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.all);
+  const [errorIsShown, setErrorIsShown] = useState(false);
```

Comment:
Better to locate `is` prefix on first place
```suggestion
  const [isError, setIsError] = useState(false);
```